explainaboard/processors/processor_registry.py

```diff
@@ -19,19 +99,6 @@ def get_processor(task: TaskType | str) -> Processor:
     Returns:
         The processor for that task
     """
-    task_cast: TaskType = TaskType(task)
-    return _processor_registry[task_cast]()
-
-
-def register_processor(task_type: TaskType):
-    """A register for task specific processors.
-
-    example usage: `@register_processor(TaskType.text_classification)`
-    """
-
-    def register_processor_fn(cls):
-        """The function to register."""
-        _processor_registry[task_type] = cls
-        return cls
-
-    return register_processor_fn
+    task_type = task if isinstance(task, TaskType) else TaskType(task)
```

Comment:
`get_processor()` looks to do nothing special. We can remove this function.

Reply:
@odashi I think we need a migration in separate PRs to remove the function because the function is exposed as a public API and it is used in the two places in explainaboard_web:
- https://github.com/neulab/explainaboard_web/blob/01b0fb4f3735f91f2d72ca8c1f00166d4e0fc071/backend/src/impl/db_utils/system_db_utils.py#L13
- https://github.com/neulab/explainaboard_web/blob/01b0fb4f3735f91f2d72ca8c1f00166d4e0fc071/backend/src/impl/default_controllers_impl.py#L12

Reply:
Could you treat this migration?

Reply:
@odashi As per our offline discussions, I'll remove `get_processor` in this PR.

Summary of the discussions:
* Updating the version of explainaboard for explainaboard_web requires some amount of work because `main` of explainaboard has breaking changes anyway.
* Migrating from `get_processor` to `get_processor_class` is straightforward. The change can be done as a part of the version update.

Reply:
Removed in commit 50d7eba4a13b25adbd5bdcdef1c7d49767850976.